ui/src/components/Map/SeverityFloatingFilter.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+      <div>
+            &ge;
+            <select
+                  :style="{ width: '70px' }"
+                  v-model="currentValue"
+                  @change="onSelectionChanged()"
+            >
+                  <option v-for="option in severities" :value="option" :key="option">{{ option }}</option>
+            </select>
+      </div>
+</template>
+
+<script>
```

Comment:
Same with this component, we can convert to the newer <script setup lang=ts> syntax and likely reduce it by a few lines.